src/main/java/net/minecraftforge/fml/common/asm/transformers/deobf/FMLRemappingAdapter.java

```diff
@@ -89,5 +102,21 @@ public void visitFieldInsn(int opcode, String originalType, String originalName,
                 mv.visitFieldInsn(opcode, type, fieldName, newDesc);
             }
         }
+
+        @Override
+        public void visitInvokeDynamicInsn(String name, String desc, Handle bsm, Object... bsmArgs)
+        {
+            // Partially taken from SpecialSource
```

Comment:
seriously unneeded, i wrote this, having this in there will just cause questions that i dont want to deal with.